components/project-list/project-list.jsx

```diff
@@ -16,28 +16,50 @@ export default React.createClass({
       displayBatchIndex: 1
     };
   },
+  getDefaultProps() {
+    return {
+      params: {
+        featured: null,
+        issue: null,
+        search: null,
+        ids: null
+      }
+    };
+  },
   componentDidMount() {
     this.fetchData(this.props.params);
   },
   componentWillReceiveProps(nextProps) {
+    // Reset this.state.entries
+    this.resetEntries();
```

Comment:
I don't think this should be called here. If I type in something new, it shouldn't clear the results while it's doing the search to find the new set; Doing that makes it feel broken because you see a moment of "0 results found", which isn't true. It's especially confusing if you're refining your search and the results you had disappear, and then the same ones reappear.